config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-periodics.yaml

```diff
@@ -120,12 +120,15 @@ periodics:
       preset-kind-volume-mounts: "true"
```

Comment:
Since we have the PR open, can you remove these labels too. We do not need `docker in docker` enabled, neither do we need any of the secrets we use for other CAPV jobs, nor the kind volume mounts.

Reply:
I was also wondering the same. But thought `docker in docker ` would be needed as usually the CI jobs create a container and inside of those container creates the containers of the repo. So thought of keeping it.